### DISCUSSION TITLE
Horizontal scrolling is not happening and Misplaced spotlight message on small screen/small div

### DISCUSSION THREAD

**Discussion by manju2495** [gilbarbara/react-joyride#849] [Q&A] [answered]
Hi,
I am trying to add joy-ride for the items inside a small div with horizontal scrolling enabled.
First item is getting spotlight and tooltip where as the same for the last button (hidden overflow -x) is not happening as expected.
Is it possible to scroll horizontally and highlight an item?
Codesandbox : https://codesandbox.io/s/suspicious-euler-km4m9n?file=/public/index.html

Also, the tooltip positioning is breaking in small screen/with small div elements.

**Comment by gilbarbara** [comment, marked as answer]
Hey @manju2495 

The library doesn't support horizontal scrolling.

**Reply by guillaume23** [reply]
Any chance it get implemented? For my project that disqualifies JoyRide.

**Reply by gilbarbara** [reply]
Not really
